DiscordBot.CoderDojoInfoModule/CoderDojoInfoModule.cs

```diff
@@ -27,15 +25,15 @@ public class CoderDojoInfoModule : ModuleBase {
                 StringBuilder response = new StringBuilder();
 
                 // the two blanks in front of linebreak are needed because discord uses Markdown
-                response.Append($"Hier die Termine für dich {Context.User.Username}:  \n"); 
+                response.Append($"Hier die Termine für dich, {Context.User.Mention}: \n"); 
                 foreach (var appointment in appointments) {
                     response.Append($"> {appointment.Date.ToString("dddd, dd.MM.yyyy", new CultureInfo("de-DE"))}  \n");
                 }
 
                 await ReplyAsync(response.ToString());
             }
             catch (Exception ex) {
-                Logger.LogError(ex, "Failed to get Appointments");
+                Logger.LogError($"Die Termine konnte nicht eingeholt werden: {ex}");
```

Comment:
Revert this one, the logger can get more useful information later when its needed out of the exception object.